src/fsharp/NameResolution.fs

```diff
@@ -3084,9 +3025,9 @@ let SuggestOtherLabelsOfSameRecordType g (nenv: NameResolutionEnv) ty (id: Ident
     labelsOfPossibleRecord
 
 let SuggestLabelsOfRelatedRecords g (nenv: NameResolutionEnv) (id: Ident) (allFields: Ident list) =
-    let suggestLabels() =
+    let suggestLabels (addToBuffer:string -> unit) =
```

Comment:
Nit: `addToBuffer:string -> unit` should have a space between colon and type, `addToBuffer: string -> unit`